src/features/events/models/virtualEvent.model.ts

```diff
@@ -0,0 +1,11 @@
+import { EventType } from "../entities/virtualEvent.entity";
+
+export class VirtualEventModel {
+    virtual_id: number;
+
+    src: string;
+
+    type: EventType;
+
+    event_id: number;
```

Comment:
No need to use the event_id in the model, later on, we will pass the ids in the *request URL parameters*